python/kwiver/vital/types/activity_type.h

```diff
@@ -0,0 +1,10 @@
+#ifndef KWIVER_VITAL_PYTHON_ACTIVITY_TYPE_H_
```

Comment:
This file does not need to exist, activity and activity_type are no longer coupled at the module level.

Reply:
Huh, okay, I missed that there is no `detected_object_type.h`...